types/gulp-connect-php/index.d.ts

```diff
@@ -0,0 +1,102 @@
+// Type definitions for gulp-connect-php 1.0
+// Project: https://github.com/micahblu/gulp-connect-php
+// Definitions by: Adam Thompson-Sharpe <https://github.com/MysteryBlokHed>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
```

Comment:
```suggestion
```
please do consider removing this line. I believe it was required only because you've referenced symbols from Node types in tests. Removing this will decrease burden of developing and testing changes in Node ecosystem.
Your test can be testing actual symbols, instead of using `console`:
```ts
connect.closeServer(result => {
    result; // $ExpectType boolean | undefined
});
```